code/controllers/subsystem/starmap.dm

```diff
@@ -143,7 +143,9 @@ SUBSYSTEM_DEF(starmap)
 			else if(in_transit_planet)
 				current_planet = to_planet
 			if(current_system.name == "Dolos") //Syndie cap
```

Comment:
Not a fan of referring to the syndicate capital directly by name, isn't there already a var for this?